python/pyspark/pandas/tests/indexes/test_reset_index.py

```diff
@@ -22,6 +22,7 @@
 from pyspark import pandas as ps
 from pyspark.testing.pandasutils import PandasOnSparkTestCase
 from pyspark.testing.sqlutils import SQLTestUtils
+from pyspark.testing.utils import assertDataFrameEqual
```

Comment:
It is not a very big deal, but I personally recommend to use `assert_frame_equal` instead of `assertDataFrameEqual` for comparing Pandas API on Spark objects because `assert_frame_equal` directly leverage the Pandas testing utils internally so it is more proper to compare the Pandas objects:

```
from pyspark.pandas.testing import assert_frame_equal
```

Reply:
See https://github.com/apache/spark/pull/43398 which introduces Pandas-like testing utils FYI.

Reply:
Got it. Changed it to `assert_frame_equal`.